spec/factories/chargeback_rate_detail.rb

```diff
@@ -102,16 +102,82 @@
     per_unit    "gigabytes"
   end
 
-  factory :chargeback_rate_detail_fixed_compute_cost, :parent => :chargeback_rate_detail do
+  factory :chargeback_rate_detail_fixed_compute_cost_1, :parent => :chargeback_rate_detail do
     description "Fixed Compute Cost 1"
     group       "fixed"
     source      "compute_1"
     per_time    "daily"
   end
 
-  factory :chargeback_rate_detail_memory_allocated_with_tiers, :parent => :chargeback_rate_detail_memory_allocated,
-                                                               :traits => [:euro, :bytes, :tiers_with_three_intervals]
+  factory :chargeback_rate_detail_fixed_compute_cost_2, :parent => :chargeback_rate_detail do
+    description "Fixed Compute Cost 2"
+    group       "fixed"
+    source      "compute_2"
+    per_time    "daily"
+  end
+
+  factory :chargeback_rate_detail_fixed_storage_cost_1, :parent => :chargeback_rate_detail do
+    description "Fixed Storage Cost 1"
+    group       "fixed"
+    source      "storage_1"
+    per_time    "daily"
+  end
+
+  factory :chargeback_rate_detail_fixed_storage_cost_2, :parent => :chargeback_rate_detail do
+    description "Fixed Storage Cost 2"
+    group       "fixed"
+    source      "storage_2"
+    per_time    "daily"
+  end
```

Comment:
Use [FactoryGirl sequences](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences) to get unique names for these columns and not duplicate a factory. 

```ruby
sequence :description do |n|
  "Fixed Storage Cost #{n}"
end
```

Should eliminate many factories below this comment.